lib/livebook_web/live/settings_live.ex

```diff
@@ -44,8 +44,46 @@ defmodule LivebookWeb.SettingsLive do
                 soon as you stop the application.
               </p>
             </div>
+
+          <!-- current version -->
+          <div class="flex flex-col space-y-8">
+            <div>
+              <h1 class="text-xl text-gray-800 font-semibold">
+                Current Version
+              </h1>
+              <p class="mt-4 text-gray-700">
+                Displays the current version of Livebook and Elixir.
+              </p>
+            </div>
+            <!-- display version -->
+            <div class="flex items-center justify-between border border-gray-200 rounded-lg p-4">
+              <div class="flex space-x-8">
+                <%= for field <- [:version, :elixir] do %>
+                    <div class="flex flex-col items-center justify-center">
+                      <h3><%= field |> Atom.to_string() |> String.capitalize() %></h3>
+                      <h4>
+                        <%= case field do %>
+                        <% :version -> %>
+                            <%= Mix.Project.config[:version] %>
```

Comment:
A heads up: we shouldn't use `Mix.Project` inside lib because `Mix` is not available in releases. I will push a fix shortly. :)